### ISSUE TITLE
Add bird expert maps

### ISSUE THREAD

**Issue by eightysteele** [andrewxhill/MOL#69]

**Comment by eightysteele** [comment]
Right, so in figuring out if we'll have time for this, we kinda need to know how soon we can get these data and also what form will they take.

**Comment by eightysteele** [comment]
Via Walter by email:

"Examples have been sitting in My Dropbox\MoL\ExpertMaps\BirdsSHP for a while. Essentially, we could produce maps in exact same format as the amphibian or mammal maps. Probably could get this done on Fri/Sat. Will want to consult with Dan here and potentially ask him to assist, to ensure consistency. Note that I am potentially envisioning two sources (Walter, IUCN) for the expert map type for birds, following the same data structure."

**Comment by walterj** [comment]
I put some examples of recent IUCN maps + metadata in My Dropbox\MoL\ExpertMaps\Birds_NewBL . Will reproject (to Mercator) and rename that before use, but would like to retain fields in dbf and display by filtering 2-3 of the fields. OK?

**Comment by eightysteele** [comment]
> Will reproject (to Mercator) and rename that before use

SGTM. Please update this issue when they are reprojected.

> Would like to retain fields in dbf and display by filtering 2-3 of the fields.

I'm not sure I understand what you mean here. What do you mean by "display by filtering 2-3 of the fields"?

**Comment by walterj** [comment]
Only display areas where field ORIGIN = 3 and PRESENCE = 1

**Comment by tucotuco** [comment]
Walter, need clarification on this issue. The files you gave us to load most recently in two big zip files named BOWmerc*.zip had no properties called ORIGIN and PRESENCE. We loaded everything that was in there to the metadata for those layers (OccCode, Latin, EditsInfo, Citation).

**Comment by walterj** [comment]
This is because these were not IUCN range maps, but Walter range maps. The expert range map type will have sources with fields that have different names and meanings. Some require several fields to characterize type of occurrence (e.g. IUCN: ORIGIN, PRESENCE, etc.), others (mine) have only one (OccCode, where 1 = breeding-only, 2= year-round resident, 3 = non-breeding only, 4 = 1 or 2).

Walter

-----Original Message-----

**Comment by walterj** [comment]
Note that I will soon (few weeks) provide updated bird maps to replace many/all of the current ones. So no need to spend much more on finalization of the current lot.
